### PR TITLE
US104432: Create a DEMO for Sort Logic Off mode.

### PR DESCRIPTION
**Pull request by njostonehouse**

### CODE REVIEW

File: demo/data/my-enrollments-embedded [hunk 1]
```diff
@@ -0,0 +1,80 @@
+{
+  "class": [
+    "enrollments",
+    "collection"
+  ],
+  "entities": [
```
Comment (njostonehouse) [inline demo/data/my-enrollments-embedded]: `embedDepth=1` is used, and required by Sort Logic Off. Took a while to sort that one out.

Comment (edwardg8) [reply demo/data/my-enrollments-embedded]: Curious what happens when it doesn't have `embedDepth=1`?

Comment (edwardg8) [reply demo/data/my-enrollments-embedded]: And I don't see how it got used/added in this PR?

**Comment by edwardg8** [comment]
Nice so good to have this demo page!

File: demo/data/my-enrollments-embedded (hunk 1 above)
Comment (njostonehouse) [reply demo/data/my-enrollments-embedded]: The demo imitates how the server will act. With the `-animated` component, all its requests have `embedDepth=1`, which means it expects the data to be organized differently. This demo has a new type of data that will work; the old data (`embedDepth=0`) wasn't compatible with the demo.

**Pull request merged**